File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_AutoTextsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_AutoTextsService : BaseService<Sys_AutoTexts, ISys_AutoTextsRepository>
    , ISys_AutoTextsService
    {
    private readonly ISys_AutoTextsRepository _Sys_AutoTextsRepository;
    public Sys_AutoTextsService(ISys_AutoTextsRepository Sys_AutoTextsRepository) : base(Sys_AutoTextsRepository)
    {
    _Sys_AutoTextsRepository = Sys_AutoTextsRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_AutoTexts.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AutoTexts } from './Sys_AutoTexts.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AutoTextsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'AutoText_Id', 'Definer_Id', 'OriginString', 'ReplaceString', 'IsAutoCorrect', 'ApplyToEveryone', 'Enabled', 'Creation_Date', 'Created_By', 'Last_Update_Date', 'Last_Updated_By'];

    List: Sys_AutoTexts[] | undefined;
    ListFilter: Sys_AutoTexts[] | undefined;
    FormData!: Sys_AutoTexts;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AutoTexts";
    }
}

